### PR TITLE
Add rss link to page template

### PR DESCRIPTION
**Pull request by dominicmkennedy**

### CODE REVIEW

**Review by bennn — CHANGES_REQUESTED** [review]
looks good but please undo all the whitespace changes

these diffs seem WAY bigger than they need to be

**Comment by dominicmkennedy** [comment]
Whoopsie, I think my editor just got a little overzealous with the auto-format. Should be fixed now

File: 2023/06/gtp-benchmarks-for-gradual-typing-performance.html [hunk 1]
```diff
@@ -52,8 +61,21 @@ <h2>GTP Benchmarks for Gradual Typing Performance</h2>
 <p>Suppose we have a typed function that averages the elements in a list:</p>
 
 <div class="brush: python">
- <pre><code>def avg(nums: list[int]):
-  ....</code></pre></div>
```
Comment (bennn) [inline 2023/06/gtp-benchmarks-for-gradual-typing-performance.html:56]: guess i should finally install pygments :)

**Review by bennn — COMMENTED** [review]
excellent, thanks!!!!

**Comment by ashton314** [comment]
RSS FTW! 🙌🏼

**Pull request merged**